### PR TITLE
[FIX] Fixes text field frame got cut off

### LINKED ISSUE #3542: Text field frame got cut off
**Describe the bug**
The text input field cuts out the bottom part of the text (settings and browser search)

**Screenshots**
https://docs.google.com/presentation/d/1eZmGHbxJk02cBE2LO3FhHPK0U-dD4XdaRsUlHepZDfU/edit#slide=id.ge2cd9c0ab9_0_9

https://docs.google.com/presentation/d/1eZmGHbxJk02cBE2LO3FhHPK0U-dD4XdaRsUlHepZDfU/edit#slide=id.ge2cd9c0ab9_0_21

**To Reproduce**
_Steps to reproduce the behavior_
A: Settings
1. Go to Settings
2. Click on General
3. Scroll down to Search Engine
4. Click on the search engine dropdown to see error

B: Browser
1. Go to browser
2. Type url to see error

**Expected behavior**
The text should show properly

**Smartphone (please complete the following information):**
 - App Version [e.g. 1.0.0] - find version number in app from Settings > About MetaMask

-------------------------------------------------------------
_to be added after bug submission by internal support / PM_
**Severity**

- How critical is the impact of this bug on a user?
- Add stats if available on % of customers impacted
- Is this visible to all users?
- Is this tech debt?

### PR DESCRIPTION
**Pull request by blackdevelopa**
**Description**

On two different screens, the text field frame cuts off the bottom part of the text. This PR fixes both bugs.

**Checklist**

* [x] There is a related GitHub issue
* [x] Tests are included if applicable
* [x] Any added code is fully documented

**Issue**

Resolves #3542

### CODE REVIEW

**Comment by blackdevelopa** [comment]
I have read the CLA Document and I hereby sign the CLA

File: package.json [hunk 1]
```diff
@@ -201,6 +201,7 @@
     "react-native-randombytes": "^3.5.3",
     "react-native-reanimated": "^1.13.2",
     "react-native-redash": "14.1.1",
+    "react-native-responsive-fontsize": "^0.5.1",
```
Comment (gantunesr) [inline package.json:204]: Can you explain why adding this dependency is necessary?

Comment (blackdevelopa) [reply package.json:204]: I introduced this package to minimize using fixed values (px) instead of responsive values that scale across different devices.

**Review by Cal-L — CHANGES_REQUESTED** [review]
Left a comment and made a PR for suggested changes:
https://github.com/MetaMask/metamask-mobile/pull/3652

File: app/components/Views/BrowserTab/index.js [hunk 2]
```diff
@@ -184,12 +185,12 @@ const styles = StyleSheet.create({
 		...fontStyles.normal,
 		backgroundColor: Device.isAndroid() ? colors.white : colors.grey000,
 		borderRadius: 30,
-		fontSize: Device.isAndroid() ? 16 : 14,
+		fontSize: Device.isAndroid() ? 16 : RFValue(13),
```
[the remark this answers is not in the document]
Comment (blackdevelopa) [reply app/components/Views/BrowserTab/index.js]: Absolutely valid point. I have updated this PR and removed the library. Thank you.

**Review by Cal-L — APPROVED** [review]
LGTM

**Pull request merged**